tests/unit/template_cassdc_test.go

```diff
@@ -33,13 +34,23 @@ var _ = Describe("Verify CassandraDatacenter template", func() {
 		err = nil
 	})
 
-	renderTemplate := func(options *helm.Options) {
-		renderedOutput := helm.RenderTemplate(
+	renderTemplateE := func(options *helm.Options) error {
+		renderedOutput, renderError := helm.RenderTemplateE(
```

Comment:
This could be simplified a bit as,

```go
		renderedOutput, err := helm.RenderTemplateE(
			GinkgoT(), options, helmChartPath, helmReleaseName,
			[]string{"templates/cassdc.yaml"},
		)

		if err == nil {
			err = helm.UnmarshalK8SYamlE(GinkgoT(), renderedOutput, cassdc)
		}

		return err
```

Because we using functions that return an error instead of panic, we need to update all the `renderTemplate` that should succeed to calls like this:

```go
Expect(renderTemplate(options)).To(Succeed())
```

Reply:
Ah yeah that looks good!
First time programming Go for me, so these patterns are a bit new :P